spec/graphql/analysis/analyze_query_spec.rb

```diff
@@ -18,8 +18,12 @@ def call(memo, visit_type, irep_node)
   describe ".analyze_query" do
     let(:node_counter) {
       -> (memo, visit_type, irep_node) {
-        memo ||= Hash.new { |h,k| h[k] = 0 }
-        visit_type == :enter && memo[irep_node.ast_node.class] += 1
+        memo ||= Hash.new
```

Comment:
I'm not mega thrilled I had to switch to `Hash.new` without the block here. But, with the block, this test (and the test below) were pushing `memo[:errors] = 0`, causing the error checking logic to break.

I could be more careful in the error checker--something like `if next_memo.is_a?(Hash) && next_memo[:errors].present? && next_memo[:errors].all? { |e| e.is_a?(GraphQL::AnalysisError}`--but opted not for the verbosity. I can do that if you think it's safer, though!